### PR TITLE
Add Rancher stack and service to structured data

### PR DESCRIPTION
**Pull request by mccricardo**
Implements `Add stack and service to logspout`.

Also includes change that does not pump full log on a restart.

### CODE REVIEW

File: router/pump.go [hunk 1]
```diff
@@ -307,6 +309,17 @@ type containerPump struct {
 }
 
 func newContainerPump(container *docker.Container, stdout, stderr io.Reader) *containerPump {
+	stack := ""
+	service := ""
+
+	if container.Config.Labels != nil {
+		if stackService, ok := container.Config.Labels ["io.rancher.stack_service.name"]; ok {
```
Comment (ruimarinho) [inline router/pump.go]: Extra space after Labels.

Comment (mccricardo) [reply router/pump.go]: Removed.

File: router/pump.go [hunk 2]
```diff
@@ -139,8 +139,10 @@ func (p *LogsPump) Run() error {
 	for event := range events {
 		debug("pump.Run() event:", normalID(event.ID), event.Status)
 		switch event.Status {
-		case "start", "restart":
-			go p.pumpLogs(event, true, inactivityTimeout)
+		case "start":
+			go p.pumpLogs(event, false, inactivityTimeout)
+		case "restart":
```
Comment (alfredopalhares) [inline router/pump.go]: Since #2 was merged, these changes can be reverted, right?

**Review by alfredopalhares — CHANGES_REQUESTED** [review]

**Review by alfredopalhares — APPROVED** [review]
LGTM

**Pull request merged**